iocore/net/SSLConfig.cc

```diff
@@ -215,40 +215,46 @@ SSLConfigParams::initialize()
 
   REC_ReadConfigInteger(client_ssl_options, "proxy.config.ssl.client.TLSv1_2");
   if (!client_ssl_options) {
-    ssl_client_ctx_protocols |= SSL_OP_NO_TLSv1_2;
+    ssl_client_ctx_options |= SSL_OP_NO_TLSv1_2;
   }
 #endif
 
 #ifdef SSL_OP_CIPHER_SERVER_PREFERENCE
   REC_ReadConfigInteger(options, "proxy.config.ssl.server.honor_cipher_order");
   if (options) {
     ssl_ctx_options |= SSL_OP_CIPHER_SERVER_PREFERENCE;
+    ssl_client_ctx_options |= SSL_OP_CIPHER_SERVER_PREFERENCE;
```

Comment:
Does this one make sense for a ATS initiating context?

Reply:
hmm from the manual, looks like it only makes sense for server.